_includes/card.html

```diff
@@ -10,14 +10,15 @@
 
```

Comment:
I checked the project and I don't see any other use of this `include` but for as a project card, I think we might consider consolidating those includes